src/core/appdir.cpp

```diff
@@ -422,9 +410,6 @@ namespace linuxdeploy {
 
                         stripOperations.insert(actualDestination);
 
-                        if (!deployElfDependencies(path, recursionLevel))
```

Comment:
This is basically the contribution, the rest is about removing the tree structure logging business.

Reply:
Thanks for highlighting, seems correct, I only see logging changes elsewhere. Let's try this out.